core/input/inputhandler.py

```diff
@@ -67,6 +75,22 @@ def __handleKeys(self):
 			Console.debug(thread="DEBUG",
 						  message=f"world.entities: {self.game.world.entities}")
 
+		# itembar
+		if keys[K_1]:
+			UI.getElementByID(1).value.setItemSlot(1)
```

Comment:
Verander `UI.getElementByID(1).value.setItemSlot()` met `UI.ITEMBAR.value.setItemSlot()`. Het saved wat memory aangezien je niet een list conversion hoeft te doen. Dit heb ik je ook eerder in een call gezegt...